capabilities/writetarget/beholder/pb/platform/error.go

```diff
@@ -0,0 +1,13 @@
+package writetarget
+
+import (
+	"fmt"
+
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+)
+
+// AsError returns the WriteError message as an (Go) error
+func (e *WriteError) AsError() error {
+	protoName := protoimpl.X.MessageTypeOf(e).Descriptor().FullName()
+	return fmt.Errorf("%s [ERR-%v] - %s: %s", protoName, e.Code, e.Summary, e.Cause)
+}
```

Comment:
Where is this used? Would it be simpler to implement error directly?
```suggestion
// AsError returns the WriteError message as an (Go) error
func (e *WriteError) Error() string {
	protoName := protoimpl.X.MessageTypeOf(e).Descriptor().FullName()
	return fmt.Errorf("%s [ERR-%v] - %s: %s", protoName, e.Code, e.Summary, e.Cause)
}
```

Reply:
It is used [here](https://github.com/smartcontractkit/chainlink-framework/blob/9d0e67205f6284751c9a751599015c5ae8f82a35/capabilities/writetarget/write_target.go#L471-L479), which needs to return an error. I changed the name to `Error()` though